tajo-catalog/tajo-catalog-drivers/tajo-hive/src/main/java/org/apache/tajo/catalog/store/HiveCatalogStore.java

```diff
@@ -845,11 +854,72 @@ public boolean existPartitionMethod(String databaseName, String tableName) throw
   }
 
   @Override
-  public List<CatalogProtos.PartitionDescProto> getPartitions(String databaseName,
-                                                         String tableName) {
-    throw new UnsupportedOperationException();
+  public List<CatalogProtos.PartitionDescProto> getPartitions(String databaseName, String tableName) {
+    throw new TajoRuntimeException(new UnsupportedException("getPartitions"));
+  }
+
+  @Override
+  public boolean existPartitions(String databaseName, String tableName) {
+    throw new TajoRuntimeException(new UnsupportedException("existPartitions"));
+  }
+
+  @Override
+  public List<PartitionDescProto> getPartitionsByAlgebra(PartitionsByAlgebraProto request) {
+    throw new TajoRuntimeException(new UnsupportedException("getPartitionsByAlgebra"));
```

Comment:
UnsupportedException automatically generates the error message including the method and class if it is used with a default constructor. SO, it should be new UnsupportedException().